iptables/iptables.go

```diff
@@ -554,6 +578,57 @@ func (ipt *IPTables) run(args ...string) error {
 	return ipt.runWithOutput(args, nil)
 }
 
+// runWithOutput runs an iptables command with the given arguments,
+// writing any stdout output to the given writer
+func (ipt *IPTables) runRestore(args []string, input string) error {
```

Comment:
I don't know if the maintainers are planning to merge this PR but it'd be nice to make this function public or maybe create a wrapper `RestoreRaw(args []string, input string)`